mmv1/templates/terraform/examples/redis_instance_mrr.tf.erb

```diff
@@ -0,0 +1,34 @@
+resource "google_redis_instance" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+  name           = "<%= ctx[:vars]['instance_name'] %>"
+  tier           = "STANDARD_HA"
+  memory_size_gb = 5
+
+  location_id             = "us-central1-a"
+  alternative_location_id = "us-central1-f"
+
+  authorized_network = data.google_compute_network.redis-network.id
+
+  redis_version     = "REDIS_6_X"
+  display_name      = "Terraform Test Instance"
+  reserved_ip_range = "192.168.0.0/29"
+  replica_count     = 5
+  read_replicas_mode = "READ_REPLICAS_ENABLED"
```

Comment:
would it be possible to also add values for readEndpoint and readEndpointPort here?

Reply:
per discussion, those are read-only.